contrib/net/http/utils.go

```diff
@@ -0,0 +1,73 @@
+package http
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/DataDog/dd-trace-go/tracer"
+	"github.com/DataDog/dd-trace-go/tracer/ext"
+)
+
+type ServeHTTP func(http.ResponseWriter, *http.Request)
+
+// Trace will trace the request before calling the ServeHTTP function passed in argument.
+func Trace(serveHTTP ServeHTTP, w http.ResponseWriter, r *http.Request, service, resource string, t *tracer.Tracer) {
+	// bail out if tracing isn't enabled
+	if !t.Enabled() {
+		serveHTTP(w, r)
+		return
+	}
+
+	// TODO: get the span from the request context
```

Comment:
@gabsn Why wasn't this TODO addressed? I can do it myself - just trying to make sure I won't miss anything.